build.sbt

```diff
@@ -91,3 +92,5 @@ lazy val root = (project in file(".")).
   settings(
     name := """security-hq"""
   )
+
+addCommandAlias("dependency-tree", "dependencyTree")
```

Comment:
Is this required because of Snyk assuming an older version of sbt?

Reply:
The command alias is required because sbt 1.0+ changes the way command line arguments are accepted.  Previously, a cli dash-delimited string was automatically converted into a camel-case string.  Apparently, sbt 1.0 drops that.